### PR TITLE
Arpalazio adapter

### PR DESCRIPTION
**Pull request by dolugen**
#282

### CODE REVIEW

File: adapters/arpalazio.js [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+
+exports.name = 'arpalazio';
+
+const baseUrl = 'http://www.arpalazio.net/main/aria/sci/annoincorso/';
+const provinceQueryPath = 'chimici/chimici.php';
+const hourlyAvgParam = 0;
+const dailyAvgParam = 3;
+const hourlyAvgPeriod = {unit: 'hours', value: 1};
+const dailyAvgPeriod = {unit: 'hours', value: 24};
+const dailyParameters = ['pm25', 'pm10'];
```
Comment (jflasher) [inline adapters/arpalazio.js]: Can I get some clarification on this daily vs hourly split? It seems like we should want hourly whenever possible? Do a quick spot check, I saw pm25 and pm10 available for Viterbo it looks like we're still grabbing the daily values?

File: sources/it.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+[
+  {
+    "url": "http://www.arpalazio.net/main/aria/sci/annoincorso/chimici.php",
+    "adapter": "arpalazio",
+    "name": "ARPALAZIO",
```
Comment (dolugen) [inline sources/it.json]: Gotcha.

File: adapters/arpalazio.js (hunk 1 above)
Comment (dolugen) [inline adapters/arpalazio.js]: We've talked about it on #282 (starting at [comment](https://github.com/openaq/openaq-fetch/issues/282#issuecomment-317225793)).

File: sources/it.json (hunk 2 above)
Comment (dolugen) [inline sources/it.json]: They're using `ARPALAZIO`, an abbreviation of "Agenzia Regionale Protezione Ambientale Del Lazio".

**Comment by dolugen** [comment]
> Also, can we put something in place that'll only grab data from a day or 2 back? Right now, it's trying to insert ~600,000 records which definitely doesn't seem necessary.

Do you want this in another branch? I was thinking you'd let this version run at least once, and then apply the recent-only change.

File: adapters/arpalazio.js (hunk 1 above)
Comment (jflasher) [inline adapters/arpalazio.js]: Oh yeah, I remember that coming up now. This approach looks good.

**Comment by jflasher** [comment]
@dolugen yeah, could you do that in another branch off this one? It's a good point about getting all the back data in place. We'll see what happens when it tries to insert 500,000 records :)

Once that other PR is in place, I'll merge this one and let it populate old data.

**Pull request merged**